### PR TITLE
Fix SR-975

### PR DESCRIPTION
**Pull request by norio-nomura**
https://bugs.swift.org/browse/SR-975

### CODE REVIEW

File: Utilities/bootstrap [hunk 1]
```diff
@@ -597,9 +597,9 @@ def main():
         embed_rpath = "$ORIGIN/../lib/swift/linux"
     else:
         embed_rpath = "@executable_path/../lib/swift/macosx"
-    cmd.extend(["-Xlinker", "-rpath", "-Xlinker", embed_rpath])
 
     cmd = [os.path.join(sandbox_path, "bin", "swift-build")]
```
Comment (davidarve) [inline Utilities/bootstrap]: Nitpicking here, but I would prefer that we move this line before the if-statement instead.

**Comment by mxcl** [comment]
Good catch, I need to do a full package build to verify. Will advise.

**Comment by mxcl** [comment]
I've confirmed it fixes the package install issue.

As assurance to the community, I'm investigating why our integration tests didn't detect this prior to submission.

Will run regular tests on this anyway, (even though they are a level before this fix will be detected, better safe than sorry).

**Comment by mxcl** [comment]
@swift-ci Please test

**Comment by norio-nomura** [comment]
Thanks! :pray:

**Pull request merged**